doc/installation_instructions/basic.rst

```diff
@@ -0,0 +1,300 @@
+.. _PyTorch: https://github.com/pytorch/pytorch
+.. _ONNX: https://github.com/microsoft/onnxruntime
+.. _RedisAI: https://github.com/RedisAI/RedisAI
+.. _freezing: https://github.com/leimao/Frozen-Graph-TensorFlow
+
+------------------------------------------------------------
+
+MacOS-only
+==========
+
+We recommend users and contributors install brew_ for managing installed
+packages.  For contributors, the following brew packages can be helpful
+
+- openmpi_ for building and running parallel SmartRedis examples
+- doxygen_ for building the documention
+- cmake_ for building SmartSim and SmartRedis from source
+
+.. _brew: https://brew.sh/
+.. _openmpi: https://formulae.brew.sh/formula/open-mpi#default
+.. _doxygen: https://formulae.brew.sh/formula/doxygen#default
+.. _cmake: https://formulae.brew.sh/formula/cmake#default
+
+For Mac OS users, the version of ``make`` that comes with the Mac command line
+tools is often 3.81 which needs to be updated to install SmartSim. Users can
+``brew install make`` to get ``make`` > 4.0 but brew will install it as
+``gmake``. An easy way around this is to do ``alias make=gmake``.
+
+.. _from-pypi:
+
+========
+SmartSim
+========
+
+There are two stages for the installation of SmartSim.
+
+ 1. `pip` install SmartSim Python package
+ 2. Build SmartSim using the `smart` command line tool
+
+Step 1: Install Python Package
+==============================
+
+Activate a new virtual environment:
+
+.. code-block:: bash
+
+    conda activate <env name>
+
+and install SmartSim from PyPi with
+the following command
+
+.. code-block:: bash
+
+    pip install smartsim
+
+If you would like SmartSim to also install Machine Learning libraries that
+can be used outside SmartSim to build SmartSim-compatible models, you
+can request their installation through the ``ml`` flag as follows:
+
+.. code-block:: bash
+
+    pip install smartsim[ml]
+    # add ray extra if you would like to use ray with SmartSim as well
+    pip install smartsim[ml,ray]
+    # or if using ZSH
+    pip install smartsim\[ml\]
+    pip install smartsim\[ml,ray\]
+
+
+At this point, SmartSim is installed and can be used for more basic features.
+If you want to use the machine learning features of SmartSim, you will need
+to install the ML backends in the section below.
+
+
+Step 2: Build SmartSim
+======================
+
+Use the ``smart`` cli tool to install the machine learning backends that
+are built into the Orchestrator database. ``smart`` is installed during
+the pip installation of SmartSim and may only be available while your
+virtual environment is active.
+
+To see all the installation options:
+
+.. code-block:: bash
+
+    smart
+
+CPU Install
+-----------
+
+To install the default ML backends for CPU, run
+
+.. code-block:: bash
+
+    # run one of the following
+    smart build --device cpu          # install PT and TF for cpu
+    smart build --device cpu --onnx   # install all backends (PT, TF, ONNX) on gpu
```

Comment:
I would maybe specify here that to run gpu u need to install CUDA then link them to instructions like above

Reply:
Good call